src/bentoml/_internal/runner/runnable.py

```diff
@@ -101,6 +103,7 @@ def method(
     def method(
         meth: t.Callable[t.Concatenate[T, P], R] | None = None,
         *,
+        stream: bool = False,
```

Comment:
fwiw I think we can infer whether the function is streaming or not via asyncgen and generator, since there is no need for this `stream` value.

Reply:
Do you think we should remove both the `stream` value for `svc.api` and `runnable.method` ? and infer from the function type?

Reply:
yes, because it is a generator or async generator then we can assume that it has to be stream. We can tell users that if it is a generator, then it should be using within streaming cases. Ofc you can write generator within sync, but that will be anti-pattern

Reply:
I agree with this from the user POV, makes the path to streaming as smooth as possible.